### ISSUE TITLE
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'min_amt / artisan migrate failed

### ISSUE THREAD

**Issue by radmedics01** [grokability/snipe-it#14083]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

After upgrading to 6.2.3, i am not able to login, because of:

Illuminate\Database\QueryException
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'min_amt' in 'where clause' (SQL: select * from `models` where `min_amt` > 0 and `models`.`deleted_at` is null) (View: /var/www/html/snipeit/resources/views/layouts/default.blade.php)

artisan migrate:status shows me this:

| Yes  | 2023_07_06_092507_add_phone_fax_to_locations                                  | 5     |
| No   | 2023_07_13_052204_denormalized_eol_and_add_column_for_explicit_date_to_assets |       |
| Yes  | 2023_07_14_004221_add_show_in_list_view_to_custom_fields                      | 5     |
| Yes  | 2023_08_01_174150_change_webhook_settings_variable_type                       | 6     |
| Yes  | 2023_08_13_172600_add_email_to_companies                                      | 6     |
| No   | 2023_08_17_202638_add_last_checkin_to_assets                                  |       |
| No   | 2023_08_21_064609_add_name_ordering_to_settings                               |       |
| No   | 2023_08_21_181742_add_min_amt_to_models_table                                 |       |
| No   | 2023_09_13_200913_fix_asset_model_min_qty_nullability                         |       |
| No   | 2023_10_25_064324_add_show_in_requestable_to_custom_fields                    |       |
| No   | 2023_12_14_032522_add_remote_ip_and_action_source_to_action_logs              |       |
| No   | 2023_12_15_024643_add_indexes_to_new_activity_report_fields                   |       |
| No   | 2023_12_19_081112_fix_language_dirs                                           |       |

It seems that i was missing a step in the last update.

Running artisan migrate shows this:

`Migrating: 2023_07_13_052204_denormalized_eol_and_add_column_for_explicit_date_to_assets

   Illuminate\Database\QueryException

  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'remote_ip' in 'field list' (SQL: insert into `action_logs` (`item_type`, `item_id`, `created_at`, `user_id`, `log_meta`, `action_type`, `remote_ip`, `user_agent`, `action_source`, `updated_at`) values (App\Models\Asset, 2217, 2023-12-31 17:40:29, ?, {"eol_explicit":{"old":0,"new":true}}, update, 127.0.0.1, Symfony, cli/unknown, 2023-12-31 17:40:29))

  at vendor/laravel/framework/src/Illuminate/Database/Connection.php:712
    708▕         // If an exception occurs when attempting to run a query, we'll format the error
    709▕         // message to include the bindings with SQL, which will make this exception a
    710▕         // lot more helpful to the developer instead of just the database's errors.
    711▕         catch (Exception $e) {
  ➜ 712▕             throw new QueryException(
    713▕                 $query, $this->prepareBindings($bindings), $e
    714▕             );
    715▕         }
    716▕     }

  • A column was not found: You might have forgotten to run your migrations. You can run your migrations using `php artisan migrate`.
    https://laravel.com/docs/master/migrations#running-migrations

      +12 vendor frames
  13  app/Models/Actionlog.php:273
      Illuminate\Database\Eloquent\Model::save()

  14  app/Observers/AssetObserver.php:67
      App\Models\Actionlog::logaction()

`

### Reproduction steps

1. php upgrade.php
2. logon

### Expected behavior

Is there a way to manually migrate the steps?

### Snipe-IT Version

6.2.3

### Operating System

Debian 11

### Web Server

Apache

### PHP Version

8.2

**Comment by radmedics01** [comment]
I did a restore to the backup before the update. Now i'm on the version:

v6.1.2 - build 10938 (master)

and artisan migrate:status shows this:

| Yes  | 2022_11_15_232525_adds_should_autoassign_bool_to_users_table             | 5     |
| Yes  | 2022_12_20_171851_fix_nullable_migration_for_settings                    | 4     |
| Yes  | 2023_01_18_122534_add_byod_to_assets                                     | 4     |
| Yes  | 2023_01_21_225350_add_eol_date_on_assets_table                           | 5     |
| Yes  | 2023_01_23_232933_add_vip_to_users                                       | 5     |
| Yes  | 2023_02_12_224353_fix_unescaped_customfields_format                      | 5     |
| Yes  | 2023_02_28_173527_adds_webhook_option_to_settings_table                  | 5     |
| Yes  | 2023_03_21_215218_update_slack_setting                                   | 5     |
| Yes  | 2023_04_12_135822_add_supplier_to_components                             | 5     |
| Yes  | 2023_04_25_085912_add_autoadd_to_customfields                            | 5     |
| Yes  | 2023_04_25_181817_adds_ldap_location_to_settings_table                   | 5     |
| Yes  | 2023_04_26_160235_add_warranty_url_to_manufacturers                      | 5     |
| Yes  | 2023_05_08_132921_increase_state_to_more_than_3                          | 5     |
| Yes  | 2023_05_10_001836_add_google_auth_to_settings                            | 5     |
| Yes  | 2023_07_05_092237_change_settings_table_increase_saml_idp_metadata_size  | 5     |
| Yes  | 2023_07_06_092507_add_phone_fax_to_locations                             | 5     |
| Yes  | 2023_07_14_004221_add_show_in_list_view_to_custom_fields                 | 5     |
| Yes  | 2023_08_01_174150_change_webhook_settings_variable_type                  | 6     |
| Yes  | 2023_08_13_172600_add_email_to_companies                                 | 6     |
+------+--------------------------------------------------------------------------+-------+

**Comment by radmedics01** [comment]
It seems that this one is missing?

| No | 2023_07_13_052204_denormalized_eol_and_add_column_for_explicit_date_to_assets | |

**Comment by felibb** [comment]
~https://github.com/snipe/snipe-it/issues/13724 is closed and supposedly fixed~ (edit: that one talks of another column, it seems), but I see the same problem when upgrading 6.1.1 to 6.2.3:

```
php artisan migrate
**************************************
*     Application In Production!     *
**************************************

 Do you really wish to run this command? (yes/no) [no]:
 > yes

Migrating: 2023_07_13_052204_denormalized_eol_and_add_column_for_explicit_date_to_assets

   Illuminate\Database\QueryException

  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'remote_ip' in 'field list' (SQL: insert into `action_logs` (`item_type`, `item_id`, `created_at`, `user_id`, `log_meta`, `action_type`, `remote_ip`, `user_agent`, `action_source`, `updated_at`) values (App\Models\Asset, 80, 2024-01-03 16:31:11, ?, {"eol_explicit":{"old":0,"new":true}}, update, 127.0.0.1, Symfony, cli/unknown, 2024-01-03 16:31:11))

  at vendor/laravel/framework/src/Illuminate/Database/Connection.php:712
    708▕         // If an exception occurs when attempting to run a query, we'll format the error
    709▕         // message to include the bindings with SQL, which will make this exception a
    710▕         // lot more helpful to the developer instead of just the database's errors.
    711▕         catch (Exception $e) {
  ➜ 712▕             throw new QueryException(
    713▕                 $query, $this->prepareBindings($bindings), $e
    714▕             );
    715▕         }
    716▕     }

  • A column was not found: You might have forgotten to run your migrations. You can run your migrations using `php artisan migrate`.
    https://laravel.com/docs/master/migrations#running-migrations

      +12 vendor frames
  13  app/Models/Actionlog.php:273
      Illuminate\Database\Eloquent\Model::save()

  14  app/Observers/AssetObserver.php:67
      App\Models\Actionlog::logaction()
```

**Comment by radmedics01** [comment]
Hi,

i already saw this issue, but what can i do at the moment to fix it?

**Comment by felibb** [comment]
@radmedics01 sorry, no clue, I just added a me-too. I'm restoring a VM backup right now. Maybe we can tag @snipe and ask for help.

**Comment by radmedics01** [comment]
Hello @snipe Team, could this be a new issue? How can we fix it?

**Comment by snipe** [comment]
I think this should be fixed in #14097
